lib/web_ui/test/channel_buffers_test.dart

```diff
@@ -23,7 +23,7 @@ void main() {
 
 ByteData _makeByteData(String str) {
   final Uint8List list = utf8.encode(str) as Uint8List;
-  final ByteBuffer buffer = list is Uint8List ? list.buffer : Uint8List.fromList(list).buffer;
```

Comment:
/cc @yjbanov I wonder in what cases we needed this `is` check; is the utf8 encoder platform-dependant somehow?

Reply:
/cc @Hixie who wrote this code and @mdebbar who LGTM'd it :)

Interestingly, why did this same check not catch the other file: https://github.com/flutter/engine/blob/master/testing/dart/channel_buffers_test.dart#L16

Reply:
/cc @scheglov had you seen https://github.com/flutter/engine/blob/master/testing/dart/channel_buffers_test.dart#L16?

Reply:
Yes, I see it now, and one more place as well. 

The repo is not clean for me, there are many classes missing, there are probably some configuration step that I missed.

Reply:
@scheglov there's quite a bunch of setup required for the engine repo, have you seen its [setup instructions](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment)?